docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -169,7 +169,7 @@ The security mapping must provide one or more configuration settings:
   credential. If list cotains "*", then any key can be specified via extra credential.
```

Comment:
Can you please format your commit message according to: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages?

Reply:
Done